src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -1311,7 +1341,18 @@ namespace winrt::TerminalApp::implementation
             SwitchToTabArgs switchToTabArgs{ idx.value() };
             action.Args(switchToTabArgs);
 
-            actions.push_back(action);
+            actions.emplace_back(std::move(action));
+        }
+
+        // If the user set a custom name, save it
```

Comment:
Hmm. So if the user has a `_quake` window that they persist, it'll open initially as a normal window, then at the end of the startup, it'll rename itself to `_quake`, and then snap to the top half.


Though, it will have also initialized itself where it was before, in FOMO and on the top half, so it's not that weird I suppose

Reply:
The only potentially weird thing is that it will have the popup that identifies the window, but I'm inclined to call that a feature.

Reply:
I actually like that too